database/ql/ql_test.go

```diff
@@ -40,23 +40,38 @@ func Test(t *testing.T) {
 		}
 	}()
 	dt.Test(t, d, []byte("CREATE TABLE t (Qty int, Name string);"))
-	driver, err := WithInstance(db, &Config{})
+}
+
+func TestMigrate(t *testing.T) {
+	dir, err := ioutil.TempDir("", "ql-driver-test")
 	if err != nil {
-		t.Fatalf("%v", err)
+		return
 	}
-	if err := d.Drop(); err != nil {
-		t.Fatal(err)
+	defer func() {
+		os.RemoveAll(dir)
+	}()
+	fmt.Printf("DB path : %s\n", filepath.Join(dir, "ql.db"))
```

Comment:
Use `t.Logf()` instead

Reply:
Right, this was a copy-pasta, I will correct it in `Test()` as well.